pkg/gardenlet/apis/config/v1alpha1/types.go

```diff
@@ -133,6 +133,9 @@ type GardenletControllerConfiguration struct {
 	// ShootCare defines the configuration of the ShootCare controller.
 	// +optional
 	ShootCare *ShootCareControllerConfiguration `json:"shootCare,omitempty"`
+	// ShootStateSync defines the configuration of the ShootStateController controller
+	// +optional
+	ShootStateSync *ShootStateSyncControllerConfiguration `json:"extensionsSync,omitempty"`
```

Comment:
According to your changes in the gardenlet helm chart and in the example configuration file this should be

```suggestion
	ShootStateSync *ShootStateSyncControllerConfiguration `json:"shootStateSync,omitempty"`
```